example/gluon/wavenet/README.md

```diff
@@ -0,0 +1,53 @@
+# WaveNet with Gluon
+
+Gluon implementation of [WaveNet: A Generative Model for Raw Audio](https://arxiv.org/abs/1609.03499)
+
+![net_structure1](assets/net_struc1.png)
+![net_structure2](assets/net_struc2.png)
+
+## Requirements
+- Python 3.6.1
+- Mxnet 1.2
+- tqdm
+- scipy.io
+
+
+## Usage
+
+- arguments
+  - batch_size : Define batch size (defualt=64)
+  - epoches : Define total epoches (default=1000)
+  - mu : Define mu (default=128)
```

Comment:
It would be clearer to give some context on what mu is as one would have to guess it's in the paper, which doesn't refer to the mu-law. E.g. Define \mu in \mu-Law (see the paper or https://en.wikipedia.org/wiki/%CE%9C-law_algorithm).